server/core/store.go

```diff
@@ -167,6 +170,11 @@ func (s *StoreInfo) IsPhysicallyDestroyed() bool {
 	return s.GetMeta().GetPhysicallyDestroyed()
 }
 
+// IsAbnormal checks if the store is abnormal.
+func (s *StoreInfo) IsAbnormal() bool {
+	return s.IsPreparing() || s.IsRemoving() || s.IsSlow()
```

Comment:
Why are preparing and removing regarded as abnormal?

Reply:
Maybe use another name ? Per se, we just want confirm that this store is not appropriate to return Messages.